patterns/hero-overlapped-book-cover-with-links.php

```diff
@@ -0,0 +1,166 @@
+												<?php echo esc_html_x( 'Shop at Apple Books', 'Hero - Overlapped book cover pattern button 2', 'twentytwentyfive' ); ?>
+											</a>
+										</div>
+										<!-- /wp:button -->
+
+									</div>
+									<!-- /wp:buttons -->
+
+								</div>
+								<!-- /wp:column -->
+
+								<!-- wp:column {"verticalAlignment":"stretch"} -->
+								<div class="wp-block-column is-vertically-aligned-stretch">
+
+									<!-- wp:buttons {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"flex","orientation":"horizontal","flexWrap":"wrap","justifyContent":"space-between"}} -->
+									<div class="wp-block-buttons">
+
+										<!-- wp:button {"backgroundColor":"contrast","textColor":"base","width":100,"className":"is-style-outline","style":{"border":{"radius":"50px"},"typography":{"fontSize":"1rem"},"elements":{"link":{"color":{"text":"var:preset|color|base"}}},"spacing":{"padding":{"top":"1rem","bottom":"1rem"}}}} -->
+										<div class="wp-block-button has-custom-width wp-block-button__width-100 has-custom-font-size is-style-outline" style="font-size:1rem">
+											<a class="wp-block-button__link has-base-color has-contrast-background-color has-text-color has-background has-link-color wp-element-button" href="#" style="border-radius:50px;padding-top:1rem;padding-bottom:1rem">
+												<?php echo esc_html_x( 'Shop at Audible', 'Hero - Overlapped book cover pattern button 3', 'twentytwentyfive' ); ?>
+											</a>
+										</div>
+										<!-- /wp:button -->
+
+										<!-- wp:button {"backgroundColor":"contrast","textColor":"base","width":100,"className":"is-style-outline","style":{"border":{"radius":"50px"},"typography":{"fontSize":"1rem"},"elements":{"link":{"color":{"text":"var:preset|color|base"}}},"spacing":{"padding":{"top":"1rem","bottom":"1rem"}}}} -->
+										<div class="wp-block-button has-custom-width wp-block-button__width-100 has-custom-font-size is-style-outline" style="font-size:1rem">
+											<a class="wp-block-button__link has-base-color has-contrast-background-color has-text-color has-background has-link-color wp-element-button" href="#" style="border-radius:50px;padding-top:1rem;padding-bottom:1rem">
+												<?php echo esc_html_x( 'Shop at Barnes & Noble', 'Hero - Overlapped book cover pattern button 4', 'twentytwentyfive' ); ?>
+											</a>
+										</div>
+										<!-- /wp:button -->
+
+									</div>
+									<!-- /wp:buttons -->
+
+								</div>
+								<!-- /wp:column -->
+
+							</div>
+							<!-- /wp:columns -->
+
+							<!-- wp:spacer {"height":"40px","style":{"layout":{"selfStretch":"fixed","flexSize":"3rem"}}} -->
+							<div style="height:40px" aria-hidden="true" class="wp-block-spacer"></div>
+							<!-- /wp:spacer -->
+
+							<!-- wp:paragraph {"metadata":{"name":"BottomText"},"style":{"typography":{"fontSize":"1.13rem","fontStyle":"normal","fontWeight":"500"}}} -->
```

Comment:
All the names on the inner blocks should be removed, to be consistent with the other patterns.

Reply:
All the block names removed.

Reply:
I am still seeing some meta data, both with and without names.
Sorry for not being clear, all meta data should be removed.

Reply:
Ok, sory for that. Fixed.